Cookbook/Technical-Documents/CRP-Bot.md

```diff
@@ -0,0 +1,99 @@
+# CRP Bot
+
+## What is the CRP process and how does the CRP bot work
+
+Because we're now certified ISO-13485, our SSDLC and the ISO process requires that we track every changes in the codebase and in each release of each flavor of our apps. This means that we need to track which commits (and which corresponding JIRA tickets) have been included in each version of the apps we push to the stores.
+
+In order to comply with that tracking requirement, every time a version of one of the flavors (Babylon UK, Bupa, Telus, Ascension US, ...) is released to one of the stores (AppStore or PlayStore), and the CRP bot is triggered (either using the Slack command – e.g. `/crp ios release/babylon/4.4.0` – or automatically during release cutoff _(soon)_) to handle that part of the release process, the bot will handle the following tasks:
+
+* Gather the CHANGELOG from the list of tickets/commits that will be included in this upcoming release.
+  - The list is built by listing the commits between the current release being cut and the last tag / GitHub release for the same flavor (e.g. between tag `telus/4.4.0` and branch `release/telus/4.5.0`) as described in our SSDLC
+  - Since each commit is annotated with a reference to the JIRA ticket it is supposed to implement, the list of commits allows to build a list of Jira tickets included in the release
+* Create the ticket on the CRP board, with the CHANGELOG (list of affected tickets) previously built
+* Create a new JIRA version on each board mentioned in the CHANGELOG (e.g. in CNSMR, NRX, AV, APPTS, … boards) and whitelisted in the bot, named like "iOS Babylon 4.3.0" (same name for each version created on each board).
+  - If a release with that exact name already exists in a board, the bot will reuse it instead of creating a new one.
+* Add that newly created version to the "Fix Version" field of the tickets found in the CHANGELOG
+
+You can find an example of [a CRP ticket dedicated to an iOS release for Babylon UK 4.4.0 here](https://babylonpartners.atlassian.net/browse/CRP-4578).
+
+> Note: for documentation about how the CRP is _implemented_ in the code of our Stevenson bot, see [this companion document](CRP-Bot-ImplementationDetails.md)._
+
+## Projects whitelist
+
+To avoid accidentally creating JIRA versions on boards others than the ones we own, or messing with boards that don't follow the same schema/template as the one expected by the bot, the CRP bot has a whitelist of boards on which it will be allowed to create JIRA versions and set Fix Versions fields.
```

Comment:
```suggestion
To avoid accidentally creating JIRA releases on boards others than the ones we own, or messing with boards that don't follow the same schema/template as the one expected by the bot, the CRP bot has a whitelist of boards on which it will be allowed to create JIRA releases and set Fix Versions fields.
```